pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -923,6 +923,8 @@ type KumaSDConfig struct {
 	// +required
```

Comment:
would it make sense to use the `URL` type which would validate that the value starts by `https?://`?